opm/simulators/wells/WellInterface.hpp

```diff
@@ -82,12 +84,7 @@ class WellInterface : public WellInterfaceIndices<GetPropType<TypeTag, Propertie
     using SparseMatrixAdapter = GetPropType<TypeTag, Properties::SparseMatrixAdapter>;
     using RateVector = GetPropType<TypeTag, Properties::RateVector>;
     using GasLiftSingleWell = ::Opm::GasLiftSingleWell<TypeTag>;
-    using GLiftOptWells = typename BlackoilWellModel<TypeTag>::GLiftOptWells;
-    using GLiftProdWells = typename BlackoilWellModel<TypeTag>::GLiftProdWells;
     using GLiftEclWells = typename GasLiftGroupInfo<Scalar>::GLiftEclWells;
-    using GLiftWellStateMap = 
-        typename BlackoilWellModel<TypeTag>::GLiftWellStateMap;
-    using GLiftSyncGroups = typename GasLiftSingleWellGeneric<Scalar>::GLiftSyncGroups;
```

Comment:
Same here: Maybe we don't need the `GLiftEclWells` type alias here, and if we do, maybe we could retrieve it from the new type instead?

Reply:
We do need the type as it's used in various well testing methods etc and I don't want to make WellInterface rely on BlackoilWellModel level stuff.

Reply:
> it's used in various well testing methods etc

Hm, pity.  I guess we'll just have to be really vigilant that the two aliases don't get out of synch.